scripts/rabbitmq-server-ha.ocf

```diff
@@ -46,6 +45,7 @@ OCF_RESKEY_erlang_cookie_file_default="/var/lib/rabbitmq/.erlang.cookie"
 OCF_RESKEY_use_fqdn_default=false
 OCF_RESKEY_fqdn_prefix_default=""
 OCF_RESKEY_max_rabbitmqctl_timeouts_default=3
+OCF_RESKEY_policy_file_default="/usr/local/sbin/set_rabbitmq_policy.sh"
```

Comment:
Okay, if folks want package it, they may use OCF param instead.
So, let's keep the change simple, which is an absolute path with a simple -f test instead of complex PATH lookups